databricks/koalas/tests/test_series_plot.py

```diff
@@ -76,12 +87,13 @@ def test_bar_plot(self):
         self.compare_plots(ax1, ax2)
 
     def test_bar_plot_limited(self):
+        self.setUpClass()
```

Comment:
This one too. Unittest will automatically call `setUpClass` when it initialises the set of tests, and `tearDownClass` when the tests are finished.

Reply:
i will remove this.